test/unit/cuda/test-forall-view.cpp

```diff
@@ -34,10 +34,9 @@ struct ForallViewCUDA : ::testing::Test {
       arr_h[i] = double(rand() % 65536);
     }
 
-    cudaErrchk(cudaMalloc((void**)&arr_d, alen * sizeof(double)));
+    cudaErrchk( cudaMalloc((void**)&arr_d, alen * sizeof(double)) );
```

Comment:
Is this a clang_format artifact?

Reply:
No, I most likely added those spaces for my own sanity because some cudaErrchk's were already in place. I may have missed 1 or 2 that are un-spaced. Is there a preference?

Reply:
I can't remember what RAJA's setup is for that, honestly I think you'd be best off just running clang-format over the whole thing and take what comes out.  If that's causing unrelated changes, you can run it over the line ranges, or just do an exemplar and match it, we have stuff like this encoded in the config file for it.

Reply:
@rchen20 note that there is a script in the scripts directory for running clang-format over the code. It's set up to avoid running it in the examples, tests, exercises directories because it screws up the formatting we want. It also avoids the tpl and blt directories because we don't want to format code we don't own.